mappings/net/minecraft/structure/pool/StructurePoolElement.mapping

```diff
@@ -3,6 +3,7 @@ CLASS net/minecraft/class_3784 net/minecraft/structure/pool/StructurePoolElement
 	METHOD method_16622 setProjection (Lnet/minecraft/class_3785$class_3786;)Lnet/minecraft/class_3784;
 		ARG 1 projection
 	METHOD method_16624 getProjection ()Lnet/minecraft/class_3785$class_3786;
+	METHOD method_16625 toRawDynamic (Lcom/mojang/datafixers/types/DynamicOps;)Lcom/mojang/datafixers/Dynamic;
```

Comment:
What does "raw" mean?

Reply:
There’re 2 `toDynamic` methods: one is protected and implemented in children, the other one is public, implemented in the parent and calls the protected one. As they have the identical signature it is necessary to give them different names, so I used `Raw` to indicate that this is not a final dynamic, which the consumer will get

Reply:
But what do they do differently? Does the public one just call the protected one, or does it also do something more?

Reply:
The public one merges some additional info with the result of the protected one

Reply:
Hmm, so like `contentToDynamic` instead?

Reply:
I don’t see much difference

Reply:
Then should be `rawToDynamic` than `toRawDynamic`

Reply:
True